assets/src/domain/eventEditor/ui/datetimes/datesList/cardView/DateCard.tsx

```diff
@@ -68,6 +65,6 @@ const DateCard: React.FC<EntityListItemProps> = React.memo(({ id }) => {
 			/>
 		</DatetimeProvider>
 	) : null;
-});
+}, getPropsAreEqual(['entity', 'cacheId']));
```

Comment:
might be a bit more readable:
`export default React.memo(DateCard, getPropsAreEqual(['entity', 'cacheId']))`

Reply:
agreed 👍

Reply:
Done